packages/editor-ui/src/components/WorkflowHistory/WorkflowHistoryListItem.vue

```diff
@@ -0,0 +1,178 @@
+<script setup lang="ts">
+import { computed, ref, onMounted } from 'vue';
+import dateformat from 'dateformat';
+import type { UserAction } from 'n8n-design-system';
+import type { WorkflowHistory, WorkflowHistoryActionTypes } from '@/types/workflowHistory';
+import type { TupleToUnion } from '@/utils/typeHelpers';
+import { useI18n } from '@/composables';
+
+const props = defineProps<{
+	item: WorkflowHistory;
+	index: number;
+	actions: UserAction[];
+	active: boolean;
+}>();
+const emit = defineEmits<{
+	(
+		event: 'action',
+		value: { action: TupleToUnion<WorkflowHistoryActionTypes>; id: WorkflowHistory['versionId'] },
+	): void;
+	(event: 'preview', value: { event: Event; id: WorkflowHistory['versionId'] }): void;
+	(event: 'mounted', value: { index: number; offsetTop: number; active: boolean }): void;
+}>();
+
+const i18n = useI18n();
+
+const actionsVisible = ref(false);
+const itemElement = ref<HTMLElement | null>(null);
+
+const formattedCreatedAtDate = computed<string>(() => {
+	const currentYear = new Date().getFullYear().toString();
+	const [date, time] = dateformat(
+		props.item.createdAt,
+		`${props.item.createdAt.startsWith(currentYear) ? '' : 'yyyy '} mmm d"#"HH:MM`,
+	).split('#');
+
+	return i18n.baseText('workflowHistory.item.createdAt', { interpolate: { date, time } });
+});
+
+const authors = computed<{ label: string; tooltip: string }>(() => {
+	const allAuthors = props.item.authors.split(', ');
+	let label = allAuthors[0];
+
+	if (allAuthors.length > 1) {
+		label = `${label} + ${allAuthors.length - 1}`;
+	}
+
+	return {
+		size: allAuthors.length,
+		label,
+	};
+});
+
+const idLabel = computed<string>(() =>
+	i18n.baseText('workflowHistory.item.id', { interpolate: { id: props.item.versionId } }),
+);
+
+const onAction = (action: TupleToUnion<WorkflowHistoryActionTypes>) => {
+	emit('action', { action, id: props.item.versionId });
+};
+
+const onVisibleChange = (visible: boolean) => {
+	actionsVisible.value = visible;
+};
+
+const onItemClick = (event: Event) => {
+	emit('preview', { event, id: props.item.versionId });
+};
+
+onMounted(() => {
+	emit('mounted', {
+		index: props.index,
+		offsetTop: itemElement.value?.offsetTop ?? 0,
+		active: props.active,
+	});
+});
+</script>
+<template>
+	<li
+		ref="itemElement"
+		:class="{
+			[$style.item]: true,
+			[$style.active]: props.active,
+			[$style.actionsVisible]: actionsVisible,
+		}"
+	>
+		<p @click="onItemClick">
+			<strong>{{ formattedCreatedAtDate }}</strong>
+			<n8n-tooltip placement="right-end" :disabled="authors.size < 2">
+				<template #content>{{ props.item.authors }}</template>
+				<span>{{ authors.label }}</span>
+			</n8n-tooltip>
+			<small>{{ idLabel }}</small>
```

Comment:
The image you attached shows truncation?

Reply:
Done by CSS